manifests/istio-telemetry/kiali/templates/kiali-operator-deployment.yaml

```diff
@@ -0,0 +1,99 @@
+{{- if .Values.kiali.operator.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kiali-operator
+  # IT IS DIRECTLY IN THE CONTROL NAMESPACE FOR NOW
+  namespace: {{ .Release.Namespace }}
```

Comment:
a different namespace would be more appropriate as discussed above.

Reply:
Yes. In fact, the way Kubernetes Operators typically are deployed is one per cluster in a standalone namespace (usually managed by OLM) and that one operator handles installing multiple operands ("operand" in this case is Kiali). This is how Kiali Operator works in multi-tenant scenarios - one operator is installed in the cluster and it can install multiple Kialis in multiple control planes/tenants.

As this PR's description states:

```
there really should only be a single operator installed in the entire cluster.
That one operator should be installed in a separate namespace within
the cluster (which istioctl should create) and that one operator's
Deployment env var WATCH_NAMESPACE should be set to "". This allows
a single cluster-wide operator to install multiple Kiali instances anywhere
in the cluster. This PR does NOT implement things this way, but it could be
changed to do that. But since this feature is considered for demo purposes
only, to keep it simple I did not implement it that way.
```

But note in my previous comment I also point out that we might not even want istioctl to have the responsibility of installing the Kiali Operator at all (require the user to have all operators installed - Prometheus, Kiali, etc. - as a pre-install step prior to using istioctl to install Istio):

```
Indeed, in the future we could rip out this entire operator-helm-chart from
istioctl and just document that the user must install the Kiali Operator already
(outside of istioctl) - at that point, istioctl need only have to create a
Kiali CR and let that externally-installed Kiali Operator install Kiali based on
the configuration found in that Kiali CR created by istioctl.
```

Reply:
I could use some help from @douglas-reid and @mandarjog here. I am struggling to understand what @istio/wg-policies-and-telemetry-maintainers wishes to maintain and support.